app/Enums/ResultStatus.php

```diff
@@ -14,4 +15,13 @@ public function getLabel(): ?string
     {
         return $this->name;
     }
+
+    public function getColor(): ?string
+    {
+        return match ($this) {
+            ResultStatus::Completed => 'success',
```

Comment:
Quick update and happy to merge, can you change `ResultStatus::` to `self::` to follow the example provided in the docs? Once it's merged we can update the column in the table to accept the enum as the available options.

https://filamentphp.com/docs/3.x/support/enums#enum-colors

Reply:
Adding `getColor` will apply color to the column out of the box, you can chain  `->badge()` to column to make it nicer, but I think only colored text is enough.